Task-1-Data-Extraction-Preprocess/books/যা_দেখেছি_যা_বুঝেছি_যা_করেছি/যা_দেখেছি_যা_বুঝেছি_যা_করেছি_1_clean.txt

```diff
@@ -0,0 +1,2157 @@
+পাকিস্তানকে শোষণ করার কথা৷ এঁ দাবির মধ্যে পাঞ্জাবী চক্র দেশদ্রোহিতার গদ্ধ
+পান তারা ৬ দফাকে বিচ্ছিন্নতাবাদী অপপ্রয়াস বলে চিত্রিত করেন৷ মূলতঃ ৬ দফা
+ছিল পাকিস্তান ভিত্তিক বাঙ্গালীদের মুক্তি সনদ মাত্র। এই ৬ দফা আন্দোলনকে স্তব্ধ
+করে দেওয়ার জন্য মুজিব ও আওয়ামী লীগের উপর বিভিন্ন হয়রানিমূলক মামলা
+চাপিয়ে দেয়৷ হয় একটার পর একটা ৷ কোন মামলায় জামিন নিয়ে জেল থেকে
+বেরুনোর মুহুর্তে জেল গেট থেকে আবার মুজিব ও তার সহকর্মীদের গ্রেফতার করা
+হত। অন্যায়ভাবে মুজিবকে এভাবে পুনঃ পুনঃ গ্রেফতার করার ফলে দেশবাসী
+বিশেষভাবে মুজিবের প্রতি আকৃষ্ট হয় ও তার প্রতি সহানুভূতিশীল হয়ে উঠে। ৬
+দফার প্রতিও জনসমর্থন বাড়তে থাকে! ক্রমান্বয়ে ৬ দফা হয়ে উঠে বাঙ্গালী জনগণের
+মনের দাবি. সোচ্চার হয়ে উঠে তারা।
+
+ঘটনার এখানেই শেষ নয়। ষড়যন্ত্র চলল বাংলার সোচ্চার কণ্ঠ দাবিয়ে রাখার । হঠাৎ
+করে ১৯৬৮ সালের ৬ই জানুয়ারী ২জন বাঙ্গালী সিএসপি অফিসার সহ মোট ২৮ জন
+সামরিক/বেসামরিক বাঙ্গালী অফিসারকে গ্রেফতার করা হয়। সরকারি প্রেসনোটে বলা
+হয় '৬৭ এর ডিসেম্বরে গ্রেফতারকৃত ব্যক্তিরা ঢাকাস্থ ভারতীয় দূতাবাসের সঙ্গে
+যোগাযোগ করে সশস্ত্র পদ্থায় পূর্ব পাকিস্তানকে আলাদা করার চক্রান্ত করছিল ৷ ১৭ই
+জানুয়ারী রাতে হঠাৎ করে মুজিবকে মুক্তি দেয়৷ হল । জেল গেট থেকে বেরিয়ে কয়েক
+পা এগুতেই সামরিক বাহিনীর ব্যক্তিরা তাকে পুনরায় গ্রেফতার করে কুর্মিটোলা
+সেনানিবাসে নিয়ে যায়! ১৮ই জানুয়ারী এক প্রেসনোটে বলা হয়, "মুজিব একটি
+ষড়যন্ত্র মামলার প্রধান আসামী” । এই মামলার নাম আগরতলা ষড়যন্ত্র মামলা ।
+আসামীর সংখ্যা ৩৫জন ৷ এর মধ্যে বঙ্গবীর এমএজি ওসমানীও রয়েছেন! রাজসাক্ষী
+১১ জন। ২১শে এপ্রিল গঠিত হল বিশেষ ট্রাইব্যুনাল ৷ বিচার শুরু হয় কুর্মিটোলা
+সেনানিবাসে ১৯শে জুন ৷ এ মামলার ঘটনার সঙ্গে শেখ মুজিব কোনভাবেই জড়িত
+ছিলেন না। তবু তাকে এ মামলায় জড়ানো হয়েছিল তার দীর্ঘ দিনের রাজনৈতিক
+জীবনের পরিসমাপ্তি ঘটানোর উদ্দেশ্যে । পাকিস্তান সামরিক বাহিনী ও প্রশাসনিক
+ক্ষেত্রে চাকুরীরত বাঙ্গালী সদস্যদের প্রায় সবাই পাঞ্জাবী শাসককুলের বিরুদ্ধে
+
+যা দেখেছি, যা বুঝেছি, যা করেছি ৬
+
+
+ 
+
+বিদ্বেষভাব পোষণ করতেন। তাদের বিজাতীয় শাসন-শোষণ, বঞ্চনা ও অবহেলার
+প্রত্যক্ষ ভুক্তভোগী ছিলেন তারা৷
+
+নৌ বাহিনীর লেফটেন্যান্ট কমান্ডার মোয়াজ্জেম সামরিক বাহিনীর বাঙ্গালীদের প্রতি
+অবিচার ও ক্রমাগত অবহেলায় পাঞ্জাবী শাসক চক্রের বিরুদ্ধে বিক্ষুব্ধ ছিলেন । স্বাধীন
+সার্বভৌম বাংলার স্বপ্ন দেখেছিলেন তিনি৷ বেঙ্গল রেজিমেন্ট, নৌ বাহিনী ও বিমান
+বাহিনীর কতিপয় দুঃসাহসী তরুণ সদস্য এবং কয়েকজন প্রভাবশালী বাঙ্গালী আমলার
+সহায়তায় তিনি পূর্ব পাকিস্তানকে বিচ্ছিন্ন করার পরিকল্পনা করেন ।
+
+স্বাধীন বাংলাদেশের চিন্তা করেছিলেন মজলুম জননেতা মাওলানা আব্দুল হামিদ খান
+ভাসানীও ৷ কিন্তু শেখ মুজিব কখনও পূর্ব পাকিস্তানকে বিচ্ছিন্ন করার কথা চিন্তা
+করেননি । তিনি স্বায়ন্ত্ব শাসনের দাবি জানিয়েছিলেন মাত্র ৷
+
+আগরতলা ষড়যন্ত্র মামলার ধারা বিবরণী খবরের কাগজে প্রকাশিত করতে থাকে
+আইয়ূব-মোনেম চক্র । তারা পূর্ব পাকিস্তানের বাঙ্গালী মুসলমানদের ভারত বিরোধী
+মনোভাবকে তাদের স্বার্থে কাজে লাগাবার উদ্দেশ্যেই এ সিদ্ধান্ত গ্রহণ করেছিলেন ৷
+পাকিস্তানের অখস্ততার বিরুদ্ধে ষড়যন্ত্রে লিপ্ত তখন তার বিরুদ্ধে বিক্ষোভে ফেটে
+পড়বে। কিন্তু পরিণাম হল সম্পূর্ণ উল্টোটাই। প্রকাশিত ধারা বিবরণীর মধ্যেই
+পরিস্ষুটিত হয়ে উঠল আইয়ুব-মোনেম চক্রের ঘৃণ্য চক্রান্ত। এই ষড়যন্ত্রমূলক
```

Comment:
পরিসষুটিত  should be পরিস্ফুটিত. In avro it will be 'porisfutito'.

Reply:
fixed